conda/conda-reqs.txt

```diff
@@ -7,10 +7,11 @@ azure-keyvault-secrets>=4.0.0
 azure-kusto-data>=4.0.0, <=5.0.0
 azure-mgmt-compute>=4.6.2
 azure-mgmt-keyvault>=2.0.0
+azure-mgmt-monitor
```

Comment:
Do we not want to use the version controls we had for pip here as well? I know azure package versioning has been a compatibility issue in the past.

Reply:
Unfortunately, when I use standard versioning numbering it fails - this package in the Conda repo uses a date-based version string. Fortunately, conda handles versioning compatibility issues better than pip so it seems to resolve itself.